charts/skypilot/templates/_helpers.tpl

```diff
@@ -31,4 +31,15 @@
   {{- fail "Error\nDeploying a SkyPilot API server requires at least 4 CPU cores and 8 GiB memory. You can either:\n1. Change `--set apiService.resources.requests.cpu` and `--set apiService.resources.requests.memory` to meet the requirements or unset them to use defaults\n2. add `--set apiService.skipResourceCheck=true` in command args to bypass this check (not recommended for production)\nto resolve this issue and then try again." -}}
 {{- end -}}
 
-{{- end -}} 
\ No newline at end of file
+{{- end -}} 
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "skypilot.serviceAccountName" -}}
```

Comment:
I'm not sure what helm convention is, but should we rename getter functions to `get....`? E.g., `skypilot.getServiceAccountName`?

Reply:
Looks like using noun (somewhat declarative) is more common in helm charts: https://github.com/kubernetes/ingress-nginx/blob/7668f45975d01b73cc20a72e88fbf0cabaf9af1d/charts/ingress-nginx/templates/_helpers.tpl#L209